### PR TITLE
Only import the lodash methods used

### PR DESCRIPTION
**Pull request by thgreasi**
Change-type: patch

### CODE REVIEW

**Comment by thgreasi** [comment]
@balena-ci rebase

File: src/api.ts [hunk 1]
```diff
@@ -120,7 +120,7 @@ export function create(req: Request): Promise<Response> {
 				.tap((res) => {
 					// Create services and associated image, labels and env vars
 					return Promise.map(
-						_.toPairs(req.composition.services),
+						toPairs(req.composition.services),
```
Comment (Page-) [inline src/api.ts:123]:
```suggestion
						Object.entries(req.composition.services),
```

File: src/api.ts [hunk 2]
```diff
@@ -219,7 +219,7 @@ function createImage(
 				)
 				.tap((releaseImage) => {
 					return Promise.map(
-						_.toPairs(labels),
+						toPairs(labels),
```
Comment (Page-) [inline src/api.ts:222]:
```suggestion
						Object.entries(labels),
```

**Comment by dfunckt** [comment]
Out of curiosity -- why @Page- ?

File: src/api.ts [hunk 3]
```diff
@@ -234,7 +234,7 @@ function createImage(
 				})
 				.tap((releaseImage) => {
 					return Promise.map(
-						_.toPairs(envvars),
+						toPairs(envvars),
```
Comment (Page-) [inline src/api.ts:237]:
```suggestion
						Object.entries(envvars),
```

**Comment by Page-** [comment]
Because loading loads of individual files takes a lot longer to start up @dfunckt, which is a big deal for cli use cases in particular. For browser use cases smaller bundle sizes are key though, and as it's all bundled in a single file the multiple file issue disappears

**Comment by thgreasi** [comment]
As discussed, closing this in favor of `Object.entries` in the next major.
See: https://github.com/balena-io-modules/balena-release/pull/15

**Pull request closed without merging**